be1-go/network/server.go

```diff
@@ -63,6 +66,8 @@ func NewServer(hub hub.Hub, addr string, port int, st socket.SocketType, log zer
 	mux := http.NewServeMux()
 	mux.HandleFunc(path, server.ServeHTTP)
 
+	mux.HandleFunc(infoPath, server.infoHandler)
```

Comment:
In principle, this is not ideal as it helps the fingerprinting of the binary for attacks.
On the other hand, at this stage, this is going to be super helpful. Thanks !